src/main/java/com/example/accountingapp/entity/User.java

```diff
@@ -0,0 +1,35 @@
+package com.example.accountingapp.entity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "users")
+@Where(clause= "is_deleted=false")
+public class User extends BaseEntity {
+
+    private String firstName;
+    private String LastName;
```

Comment:
if u change LastName to lastName will be great ...actually they are not camel case in ER...thank u

Reply:
oh, yes. Thank you @MehmetKaldirim .  
I actually updated it, but then lost all my changes after I pulled updates from master, and then i did not notice this again. Will update today and push again.